webcompat/templates/shared/footer.html

```diff
@@ -40,33 +39,33 @@
       </ul>
 
 <!-- sub nav for contribute > working with bugs -->
-      {% if request.url_rule.endpoint in ['contributors_report_bug', 'contributors_reproduce_bug', 'contributors_diagnose_bug', 'contributors_site_outreach'] %}
+      {% if 'contributors_bug_' in request.url_rule.endpoint %}
```

Comment:
For this, using `rule`, I would have to look for `-bug` and `site-outreach`, rather than just `contributors_bug_`